Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -21,97 +32,79 @@ public struct RenderSettingsData
 	public TimeRenderSettings nightSettings;
 
 	public Material skyMaterial;
+
+	public static RenderSettingsData Lerp(RenderSettingsData fromSettings, RenderSettingsData toSettings, float t)
+	{
+		RenderSettingsData returnSettings = new RenderSettingsData();
+
+		returnSettings.daySettings = TimeRenderSettings.Lerp( fromSettings.daySettings, toSettings.daySettings, t );
+		returnSettings.nightSettings = TimeRenderSettings.Lerp( fromSettings.nightSettings, toSettings.nightSettings, t );
+		returnSettings.skyMaterial = ( t < 0.5f ) ? fromSettings.skyMaterial : toSettings.skyMaterial;
+
+		return returnSettings;
+	}
 }
 
 [ExecuteInEditMode]
 public class RenderSettingsManager : SingletonBehaviour<RenderSettingsManager> 
 {
 	[SerializeField] RenderSettingsData _currentRenderSettings = new RenderSettingsData();
-	RenderSettingsData _targetRenderSettings = new RenderSettingsData();
+	[SerializeField] TimeRenderSettings _currentTimeRenderSettings = new TimeRenderSettings();
 
 	[SerializeField] Light _dirLight = null;
-	Material _curSkyboxMaterial = null;
 	int _curSkyboxTintPropertyID = 0;
 
-	[Tooltip("The length (in seconds) of a day")]
-	[SerializeField] float _dayCycleTime = 60;
-	[SerializeField] float _dayCycleTimer = 0f;
-
-	[Tooltip("Zero is noon, 1 is midnight")]
-	[Range(0, 1)]
-	public float timeOfDay; // This should not go in here. Eventually it will be with the Day Cycle
+	[ReadOnly("Daylight Intensity"), Tooltip( "0 is midnight. 1 is noon." )]
+	[SerializeField]
+	float _daylightIntensity = 1.0f;
+	public float daylightIntensity
+	{
+		get
+		{
+			return _daylightIntensity;
+		}
+		private set
+		{
+			_daylightIntensity = value;
+		}
+	}
 
 	void Awake()
 	{
-		_curSkyboxMaterial = _currentRenderSettings.skyMaterial;
 		_curSkyboxTintPropertyID = Shader.PropertyToID( "_Tint" );
 	}
 
 	void Update()
 	{
-		_dayCycleTimer += Time.deltaTime;
+		daylightIntensity = Mathf.Cos( DayCycleManager.dayCycleTimer / DayCycleManager.dayCycleLength * 2 * Mathf.PI ) * -0.5f + 0.5f;
 
-		timeOfDay = ( -Mathf.Cos( _dayCycleTimer/_dayCycleTime * 2f * Mathf.PI ) * 0.5f ) + 0.5f;
+		_currentTimeRenderSettings = TimeRenderSettings.Lerp( _currentRenderSettings.nightSettings, _currentRenderSettings.daySettings, daylightIntensity );
```

Comment:
The change to daylightIntensity rather than timeOfDay makes sense and is interesting. Do you think the concept of night/day time settings still make sense or should it be light/dark or something else?